tests/test_bgbb_udfs.py

```diff
@@ -0,0 +1,78 @@
+from lifetimes.datasets import load_donations
+import pyspark
+from pytest import fixture
+
+from bgbb import BGBB
+from bgbb.sql.bgbb_udfs import mk_udfs
+
+spark = pyspark.sql.SparkSession.builder.appName("test").getOrCreate()
+df = fixture(load_donations)
+bgbb = fixture(lambda: BGBB(params=[1.20, 0.75, 0.66, 2.78]))
+params = fixture(lambda: [1.20, 0.75, 0.66, 2.78])
+
+
+def test_mk_udfs(df, bgbb, params):
+    dfs = spark.createDataFrame(df)
+
+    params = [1.20, 0.75, 0.66, 2.78]
+    bgbb = BGBB(params=params)
+
+    p_alive, n_returns = mk_udfs(
+        bgbb, params=params, return_in_next_n_days=14, alive_n_days_later=0
+    )
+    dfs2 = (
+        dfs.withColumn("P14", n_returns(dfs.frequency, dfs.recency, dfs.n))
+        .withColumn("P_alive", p_alive(dfs.frequency, dfs.recency, dfs.n))
+    )
+    pa = [
```

Comment:
How did you generate these values? A brief comment would be useful.

Reply:
I'll change it so that the reference values are generated by lifetimes